Dockerfile

```diff
@@ -30,19 +30,9 @@ ARG NPM_BASE_64_AUTH
 ARG NPM_EMAIL
 ENV DEBIAN_FRONTEND=noninteractive
 
-# install legacy build environment for docs
-RUN apt-get -o Acquire::Check-Valid-Until=false update
-RUN apt-get -y install libpython2.7-dev python2.7 git rsync unzip curl
-RUN curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py
-RUN python2.7 get-pip.py
-RUN pip install requests virtualenv virtualenvwrapper py-dateutil
-RUN python2.7 -m pip install python-dateutil
-RUN virtualenv /venv
-RUN /venv/bin/pip install --upgrade --force setuptools
-RUN /venv/bin/pip install -r https://raw.githubusercontent.com/mongodb/docs-tools/master/giza/requirements.txt
 
 # helper libraries for docs builds
-RUN apt-get update && apt-get install -y vim git
+RUN apt-get update && apt-get install -y vim git unzip rsync curl
 
 
 ENV PATH="${PATH}:/opt/snooty:/opt/mut:/home/docsworker-xlarge/.local/bin:/usr/local/lib/python2.7/dist-packages/virtualenv/bin"
```

Comment:
NIT: We can update this string to not include `/usr/local/lib/python2.7/dist-packages/virtualenv/bin`

Reply:
Ok! Should I just truncate it after `docsworker-xlarge` or replace the entire thing with something else?

Reply:
I would truncate it after `/home/docsworker-xlarge/.local/bin` if that is what you mean. It should look like: `${PATH}:/opt/snooty:/opt/mut:/home/docsworker-xlarge/.local/bin`